lfs.h

```diff
@@ -9,6 +9,7 @@
 
 #include <stdint.h>
 #include <stdbool.h>
+#include <lfs_util.h>
```

Comment:
Oh I was surprised this wasn't already included. Does this need to use angle-brackets?